sql/mob_spawn_points.sql

```diff
@@ -3760,7 +3760,7 @@ INSERT INTO `mob_spawn_points` VALUES (16851248,'Gorger','Gorger',20,15.474,-0.5
 
 -- TODO: Implement Apex Mobs
 
-INSERT INTO `mob_spawn_points` VALUES (16851267,'Satiator','Satiator',21,-205.880,-0.500,268.850,80);
+INSERT INTO `mob_spawn_points` VALUES (16851267,'Satiator','Satiator',21,-44.3170,0,-242.987,80);
```

Comment:
Didn't really matter which one was the "default". on despawn as long as the function `UpdateNMSpawnPoint()` is called it'll select a new one from the other table.

You will need to edit [the mobs script](https://github.com/LandSandBoat/server/blob/base/scripts/zones/Promyvion-Dem/mobs/Satiator.lua) or your extra row in NM Spawn Points will do nothing, as I see the script currently does not call the function. If you want the original spawn at server start to pick a random place, you'll have to do that in the zone script using `setSpawn()` to set its "home" then `setPos()` to move it there (otherwise it will walk back to the 1st position)

Reply:
So the bigger issue is the MR portals will NOT send players to other island (where the NM is located at). By changing it to the later, the NM spawns on the island where players are transported. Perhaps comment out the other one until the portals are fixed?

Reply:
yes, commented out is fine, but honestly, I would have focused on why the portal isn't random-ing before touching the NM. The problem isn't the NMs problem, and this is working around an issue without really fixing it. So if #1250 is being closed by this I expect a new issue for the teleport problem. A whole island is inaccessible atm and we're kind of removing the incentive to fix it now. I say kind of because I dunno anyone that seriously camps this thing >.>;

Reply:
I understand. This is a temporary fix for a larger problem as stated. Either way, the NM does have 2 spawn points and until the portals are fixed, this would at least make the NM available. I suppose the NM is trivial at post 75, but its pretty useful nonetheless.

Reply:
which problem is larger is debatable and honestly not important right now as it's already agreed that it's fine to comment it out - just make a new tracker issue for the portal please. Otherwise it'll be forgotten.

Reply:
Sounds good.

Reply:
Well hold on, out of curiosity, I went and looked at the other 2 3rd floor NM's. Appears they too, spawn on the inaccessible island. let me update the other two to put on the accessible island and then can create a more specific issue for teleporters.